crowbar_framework/app/controllers/application_controller.rb

```diff
@@ -178,4 +187,25 @@ def enforce_installer
       end
     end
   end
+
+  def sanity_checks_cache
+    respond_to do |format|
+      format.html do
```

Comment:
Do we really need a `respond_to` here? If yes, why don't we handle the json case?